querybook/server/logic/comment.py

```diff
@@ -0,0 +1,147 @@
+from querybook.server.app.db import with_session
+from models.comment import Comment, CommentReaction, DataTableComment, DataCellComment
+from querybook.server.const.comment import CommentDict
+
+
+@with_session
+def get_comment_by_id(comment_id: int, session=None):
+    return Comment.get(id=comment_id, session=session)
+
+
+@with_session
+def get_all_comment_data_dict_by_id(comment_id: int, session=None) -> CommentDict:
+    comment = Comment.get(id=comment_id, session=session)
+    child_comments = (
+        session.query(Comment)
+        .filter(Comment.parent_comment_id == comment_id)
+        .order_by(Comment.created_at.asc())
+        .all()
+    )
+    reactions = (
+        session.query(CommentReaction)
+        .filter(CommentReaction.comment_id == comment_id)
+        .all()
+    )
+    comment_dict = comment.to_dict()
+    if child_comments:
+        comment_dict["child_comments"] = [
+            child_comment.to_dict() for child_comment in child_comments
+        ]
+    comment_dict["reactions"] = [reaction.to_dict() for reaction in reactions]
+
+    return comment_dict.sort(key=lambda element: element["created_at"])
```

Comment:
`coment_dict` is a `dict`, I dont think it has `sort` method.

if you're trying to sort the child comments, it's already ordered by created_at